perl-watch-cron-logs/watch_cron_logs3.pl

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/env perl
```

Comment:
This PR is just using the "copy the already-made" file for now; if we need to go to your preference to use the git repos as a reference with continuous updates, this can move into the skare3 repo... or in fact all of skare3-perl can move in there too.